pkgs/development/libraries/aribb25/default.nix

```
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitLab, fetchpatch, autoreconfHook, pkgconfig, pcsclite
+, PCSC }:
```

Same as above, also eases appending more later:
```suggestion
, PCSC
}:
```